### ISSUE TITLE
Mastodon icon?

### ISSUE THREAD

**Issue by cybeardjm** [dshanske/syndication-links#207]
Being on masto.ai, the icon pointing to a post on this instance appears as a generic "website" icon on my 2 blogs.
Not sure if I dreamt it or not, but I visualize a Mastodon icon in the past...
Am I wrong or is there a way to fix this?
Thx

**Comment by dshanske** [comment]
It has to know it is a Mastodon site..There are just too many to figure out automatically..Let me see about specifying it.

**Comment by cybeardjm** [comment]
Yes, of course, I understand. Could be a field where the plugin is told what instance(s) to look for?

**Comment by dshanske** [comment]
That's what I was thinking. Should be easy enough.

**Comment by dshanske** [comment]
I added something in a previous version that might help. If there is a mastodon URL associated with any of the users, it will use the mastodon icon.

The IndieWeb plugin, for examples, allows a mastodon URL to be set.

**Comment by nsmsn** [comment]
I switched to a new instance of Mastodon this week (https://social.lol) and am trying to figure out how to get the mastodon icon to show up. 

Is there a setting in IndieWeb plugin to set a mastodon URL? Where do I find this?
